composer.json

```
@@ -18,9 +18,9 @@
     }
   ],
   "require": {
-    "php": ">=7.1",
+    "php": "^7.4 || ^8.0",
     "contao/core-bundle": "^4.9",
-    "doctrine/dbal": "^2.12"
+    "doctrine/dbal": "^3.3"
```

We need to stay compatible with dbal in the 2.* branch:
```suggestion
    "doctrine/dbal": "^2.12 || ^3.3"
```